src/builder.rs

```diff
@@ -1,33 +1,51 @@
-use blsttc::{PublicKeySet, SignatureShare};
-use std::collections::{BTreeSet, HashMap, HashSet};
+use blsbs::{Envelope, Fr, SignedEnvelopeShare, SlipPreparer};
+use blsttc::{IntoFr, PublicKeySet, SecretKeyShare, Signature, SignatureShare};
+use std::collections::{BTreeMap, BTreeSet, HashMap, HashSet};
 use std::iter::FromIterator;
 
-use curve25519_dalek_ng::scalar::Scalar;
-
 use crate::{
-    Amount, AmountSecrets, Dbc, DbcContent, Error, NodeSignature, PublicKey, ReissueShare,
-    ReissueTransaction, Result, SpendKey,
+    Amount, Dbc, DbcContent, DbcEnvelope, Denomination, Error, PublicKey, ReissueRequest,
+    ReissueShare, ReissueTransaction, Result, SpendKey,
 };
 
 ///! Unblinded data for creating sn_dbc::DbcContent
+#[derive(Debug, Clone)]
 pub struct Output {
-    pub amount: Amount,
+    pub denomination: Denomination,
     pub owner: PublicKey,
```

Comment:
This looks like it's just the DbcContent, we can probably remove this `Output` struct now